tests/dryrun/v3_install_test.go

```diff
@@ -1,15 +1,25 @@
 package dryrun
 
 import (
+	_ "embed"
 	"os"
 	"path/filepath"
 	"testing"
 
 	"github.com/replicatedhq/embedded-cluster/pkg/dryrun"
+	"github.com/replicatedhq/embedded-cluster/pkg/release"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	//go:embed assets/install-license-valid-signature.yaml
```

Comment:
why not just replace the current license we use in dryrun tests and update related assertions? it's going to be confusing to know when to use which license.

Reply:
and also update this line in the new release file with an actual version label:
```
versionLabel: "fake-version-label"
```